CHANGELOG-UNRELEASED.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Changes `LinkAdd` and `RemoveEntry` so that they return a hash instead of a null [#1343](https://github.com/holochain/holochain-rust/pull/1343)
+- Updated linked [n3h](https://github.com/holochain/n3h) version to v0.0.12-alpha
```

Comment:
Here we should expand this with minor details re. 
- persistent transport Ids
- MDNS

Reply:
+1